infrastructure/cdn-in-a-box/cache/run.sh

```diff
@@ -17,14 +17,78 @@
 # specific language governing permissions and limitations
 # under the License.
 
-set -e
-set -x
-set -m
+set -o errexit -o xtrace -o monitor
+
+mkdir /tmp/ort
+
+set-dns.sh
+insert-self-into-dns.sh
+
+source /to-access.sh
+
+# Wait on SSL certificate generation
+until [[ -f "$X509_CA_ENV_FILE" ]]
+do
+  echo "Waiting on Shared SSL certificate generation"
+  sleep 3
+done
+
+# Source the CIAB-CA shared SSL environment
+until [[ -n "$X509_GENERATION_COMPLETE" ]]
+do
+  echo "Waiting on X509 vars to be defined"
+  sleep 1
+  source "$X509_CA_ENV_FILE"
```

Comment:
Inconsistent indentation using 2 spaces - existing code in this file used 4 spaces. I wouldn't object to switching to tabs, but it should be done consistently and globally through the file

Reply:
Spacing made consistent in 4d2c0da326.